<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Patient - Home Page</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="leftFrame">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(51, 209, 122);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="welcomePatientLabel">
          <property name="styleSheet">
           <string notr="true">font: 100 italic 32pt &quot;Ubuntu Sans&quot; ;
color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Let's get you
 FIT!</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="rightFrame">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QGroupBox" name="BMIGroupBox">
            <property name="maximumSize">
             <size>
              <width>2000</width>
              <height>1000</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="title">
             <string>Calculate BMI</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="heightLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Height</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="heightComboBox">
                     <property name="styleSheet">
                      <string notr="true">color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                     <property name="insertPolicy">
                      <enum>QComboBox::InsertAtBottom</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="mHeightLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>m</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="weightLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Weight</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="weightComboBox">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>true</bool>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                     <property name="duplicatesEnabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="kgWeightLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>kg</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="calcBMIButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Calculate</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="rightMargin">
                    <number>120</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="BMILabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Your BMI</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="BMIResultLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>BMI Result</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>75</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="BMIConditionLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Condition</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="BMIConditionResultLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Condition Result</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QGroupBox" name="bloodPressureGroupBox">
            <property name="maximumSize">
             <size>
              <width>2000</width>
              <height>1000</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="title">
             <string>Blood Pressure Analysis</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_7">
                   <item>
                    <widget class="QLabel" name="systolicLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Systolic pressure</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="systolicComboBox">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="upperNumLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Upper Number</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QLabel" name="diastolicLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Diastolic pressure </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="diastolicComboBox">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="editable">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="lowrLabel">
                     <property name="styleSheet">
                      <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                     </property>
                     <property name="text">
                      <string>Lower Number</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="analyzeBPButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Analyze</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <property name="rightMargin">
                  <number>50</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="bloodPressureConditionLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string>Condition</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="bloodPressureConditionResultLabel">
                   <property name="styleSheet">
                    <string notr="true">font: 100 italic 13pt &quot;Ubuntu Sans&quot; ;
color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="text">
                    <string>Condition Result</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="minimumSize">
             <size>
              <width>700</width>
              <height>370</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="title">
             <string>Diet Plan</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="createNewDPButton">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>200</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Create new Diet Plan</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="dietPlanList"/>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <property name="leftMargin">
                  <number>250</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QComboBox" name="feedBackcomboBox">
                   <property name="maximumSize">
                    <size>
                     <width>75</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <item>
                    <property name="text">
                     <string>1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>3</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>4</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>5</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QPushButton" name="giveFeedbackButton">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>50</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>100</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Give Feedback</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
